TiDB Integration: user experience improvements and 2.1.0 release (#1094)
DataDog/integrations-extras: tidb/README.md

> **Note:** 
>
> - TiDB 4.0+ is required for this integration. 

> > - For TiDB Cloud, please refer to [TiDB Cloud Integration][12].

Reviewer: ```suggestion
> - For TiDB Cloud, refer to the [TiDB Cloud Integration][12].
```

See [metadata.csv][6] for a list of metrics provided by this check.

> > It is possible to use the `metrics` config to collect other metrics from a TiDB cluster.

Reviewer 2: ```suggestion
> It is possible to use the `metrics` configuration option to collect additional metrics from a TiDB cluster.
```

> > It is possible to use the `metrics` config to collect more other metrics from a TiDB cluster.

Reviewer: ```suggestion
> It is possible to use the `metrics` config to collect other metrics from a TiDB cluster.
```

## Troubleshooting

### Missing CPU and Memory metrics for TiKV and TiFlash instances on macOS

> You will not get CPU or Memory metrics for TiKV and TiFlash instances in two cases:
> 
> - Running TiKV or TiFlash instances with [tiup playground][13] on macOS.
> - Or, running TiKV or TiFlash instances with [docker-compose up][14] on a new Apple M1 machine.

Reviewer: ```suggestion
CPU and Memory metrics are not provided for TiKV and TiFlash instances in the following cases:

- Running TiKV or TiFlash instances with [tiup playground][13] on macOS.
- Running TiKV or TiFlash instances with [docker-compose up][14] on a new Apple M1 machine.
```

[9]: https://docs.datadoghq.com/agent/kubernetes/log/
[10]: https://docs.datadoghq.com/agent/guide/community-integrations-installation-with-docker-agent
[11]: https://github.com/DataDog/integrations-extras/blob/master/tidb/assets/service_checks.json
[12]: https://docs.datadoghq.com/integrations/tidb_cloud/
[13]: https://docs.pingcap.com/tidb/stable/tiup-playground

> [14]: 

Reviewer 2: Can you add this link?
